examples/conanfile/package_id/settings_and_options.rst

```diff
@@ -0,0 +1,242 @@
+.. _examples_conanfile_package_id_settings_options:
+
+How settings and options of a recipe influence its package ID
+=============================================================
+
+In Conan, a package ID is a unique identifier for a package binary that takes into account all the factors that affect its contents.
```

Comment:
Maybe something like this would be more accurate?
```suggestion
In Conan, a package ID is a unique identifier for a package binary that takes into account all the factors that affect its binary compatibility.
```